docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -0,0 +1,1001 @@
+### ParametricPulse
+
+The `qiskit.pulse.library.parametric_pulses.ParametricPulse` base class and
+pulse library have been superseded by `qiskit.pulse.SymbolicPulse` and the
+corresponding pulse library. `SymbolicPulse` provides better performance,
+flexibility, and [QPY](../qiskit/qpy) support:
+
+Deprecated                       |   Alternative
+---------------------------------|--------------------------------------------
+[`pulse.library.parametric_pulses.ParametricPulse`](/api/qiskit/0.46/qiskit.pulse) | [`qiskit.pulse.SymbolicPulse`](/api/qiskit/qiskit.pulse.SymbolicPulse)
+[`pulse.library.parametric_pulses.Constant`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`pulse.library.parametric_pulses.Drag`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`pulse.library.parametric_pulses.Gaussian`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`qiskit.pulse.library.parametric_pulses.GaussianSquare`](/api/qiskit/0.46/qiskit.pulse) |  ?
+
+<span id="injecting-circuit-gate-operations"></span>
+### Injecting circuit gate operations
+
+Injecting circuit gate operations into the pulse builder context through
+`qiskit.pulse.builder.call` is no longer possible. This removal affects input
+arguments of type `QuantumCircuit` as well as the following functions:
+
+* `qiskit.pulse.builder.call_gate`
+* `qiskit.pulse.builder.cx`
+* `qiskit.pulse.builder.u1`
+* `qiskit.pulse.builder.u2`
+* `qiskit.pulse.builder.u3`
+* `qiskit.pulse.builder.x`
+
+If you still wish to inject backend-calibrated schedules, use the following
+pattern instead of calling gate commands.
+
+```python
+from qiskit.providers.fake_provider import GenericBackendV2
+from qiskit import pulse
+
+backend = GenericBackendV2(num_qubits=5)
+sched = backend.target['x'][(qubit,)].calibration
+
+with pulse.build() as only_pulse_scheds:
+    pulse.call(sched)
+```
+
+Similarly, a `QuantumCircuit` can be injected in the builder context by
+manually transpiling and scheduling the object.
+
+```python
+from math import pi
+from qiskit.compiler import schedule, transpile
+qc = QuatumCircuit(2)
+qc.rz(pi/2, 0)
+qc.sx(0)
+qc.rz(pi/2, 0)
+qc.cx(0, 1)
+qc_t = transpile(qc, backend)
+sched = schedule(qc_t, backend)
+with pulse.build() as only_pulse_scheds:
+    pulse.call(sched)
+```
+
+We recommend writing a minimum pulse program with the builder
+and attach it to `QuantumCircuit` through the `QuantumCircuit.add_calibration`
+method as a microcode of a gate instruction, rather than writing the entire
+program with the pulse model.
+
+<span id="builder.build"></span>
+### builder.build
+
+The following arguments in `qiskit.pulse.builder.build` have been removed:
+
+* `default_transpiler_settings`
+* `default_circuit_scheduler_settings`
+
+Together with the functions:
+
+* `qiskit.pulse.builder.active_transpiler_settings`
+* `qiskit.pulse.builder.active_circuit_scheduler_settings`
+* `qiskit.pulse.builder.transpiler_settings`
+* `qiskit.pulse.builder.circuit_scheduler_settings`
+
+The alternative is ?
```

Comment:
Same comment as above, we can maybe look for the alternatives in a follow-up.

Reply:
No alternative is provided because it's no longer possible to inject circuit object into the builder context and these are the setting for converting injected object into pulse representation behind the scene.